metsrw/metadata.py

```diff
@@ -179,9 +179,11 @@ def parse(cls, root):
         obj.status = status
         return obj
 
-    def serialize(self, now):
+    def serialize(self, now=None):
         created = self.created or now
```

Comment:
This isn't new, but I just noticed that the `now` value can be silently ignored? Makes more sense with `now` being able to be `None` but I wonder if this might still be unexpected?